content/python/concepts/functions/terms/parameters/parameters.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Parameters'
+Description: ''
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+Functions often need outside data to achieve their intended task. This is where parameters can be used as placeholders for that data.
+
+Parameters are variables that are declared in the function definition. They are usually processed in the function body to produce the desired result. When the function is called, each parameter is assigned the value which was passed as a corresponding argument.
+
+For example, the function below contains parameters for a `character`, a `setting`, and a `skill`, which are used as inputs to write the first sentence of a book.
+
+```py
+def write_a_book(character, setting, skill):
+  print(character + " is in " +
+        setting + " practicing " +
+        skill + ".")
+
+write_a_book("Naomi", "engineering school", "welding")
+```
+
+The output will look like this:
+
+```shell
+Naomi is in engineering school practicing welding.
+```
+
+## Default Parameter Values
```

Comment:
I might roll this into the `Arguments` term. Since it already covers most of the other idiosyncrasies of defining parameters in a function.